sig-community-experience/meeting-template.md

```diff
@@ -0,0 +1,31 @@
+# Name of the Group/Governance Entity
+
+## Meta Information
+### Links, Resources, etc
+
+## Date & Time | Title in Calendar
+
+### Pre-meeting area
+#### Headline
+#### Main Topics to be discussed
+#### People/Teams/Skills needed
+#### Preparation
+#### Links to agenda, notes, slides, …
+
+### During meeting
+#### Attendees
+#### Agenda
```

Comment:
I know we've been using this for a while now, but it just came to mind: conceptually, doesn't the *Agenda* belong to the *Pre-meeting* area?

I understand that the difference between *Main Topics to be discussed* there and *Agenda* here is about the level of detail (right?), I'm just wondering how to make that crystal clear...

Maybe a(nother) comment?

Reply:
@codificat 
in general: yes
but this pre-meeting area so far is only used for the Pre-NewsFlash and not really part of the Meeting notes on the long run in my opinion.
As soon as we figured out how to handle the NewsFlash finally and creating an abstract without the help of this pre-meeting area, we can use this in another way or get rid of it.

So for now: the Agenda should stay in the During Meeting chapter